addons/web/static/tests/core/browser/title_service.test.js

```diff
@@ -0,0 +1,58 @@
+/** @odoo-module */
+
+import { makeMockEnv, getService } from "@web/../tests/web_test_helpers";
+import { expect, test, beforeAll, beforeEach } from "@odoo/hoot";
+
+let titleService;
+
+beforeAll(async () => {
+    await makeMockEnv();
+    titleService = await getService("title");
+});
+
+beforeEach(() => {
+    document.title = "";
+    titleService.setParts({ one: null, two: null, three: null });
+});
+
+test("simple title", () => {
+    titleService.setParts({ one: "Odoo" });
+    expect(titleService.current).toBe("Odoo");
+});
+
+test("add title part", () => {
+    titleService.setParts({ one: "Odoo", two: null });
+    expect(titleService.current).toBe("Odoo");
+    titleService.setParts({ three: "Import" });
+    expect(titleService.current).toBe("Odoo - Import");
+});
+
+test.debug("modify title part", () => {
```

Comment:
debug :-)

Reply:
wow, and it's not spotted by the python test. We must fix/add the python test right now. @SimonGenin Could you do it?

Reply:
ok, on it